src/inventory/inventory.resolver.ts

```diff
@@ -9,11 +9,11 @@ import { UUID } from 'src/shared/scalars/CustomUuidScalar';
 export class InventoryResolver {
   constructor(private readonly inventoryService: InventoryService) {}
 
-  @Mutation(() => Inventory)
+  @Mutation(() => [UUID])
```

Comment:
Returning only the UUID is uncommon in GraphQL, as typically the user decides what to get (and can also limit the result)
https://blog.logrocket.com/anti-patterns-graphql-schema-design/